src/commands/serve.ts

```diff
@@ -18,17 +21,40 @@ export const desc: string = 'Serve ad4m service at given port';
 export const builder = (yargs: Argv) =>
   yargs
     .options({
-      port: { type: 'number', describe: 'Use this port to run ad4m GraphQL service', default: 4000, alias: 'p' },
-      hcAdminPort: { type: 'number', describe: 'Admin port of holochain conductor' },
-      hcAppPort: { type: 'number', describe: 'Port used by hApp' },
-      connectHolochain: { type: "boolean", describe: 'Flag to connect existing running holochain process'}
+      port: { 
+        type: 'number', 
+        describe: 'Use this port to run ad4m GraphQL service', 
+        default: 4000, 
+        alias: 'p'
+      },
+      hcAdminPort: { 
+        type: 'number', 
+        describe: 'Admin port of holochain conductor'
+      },
+      hcAppPort: { 
+        type: 'number', 
+        describe: 'Port used by hApp' 
+      },
+      connectHolochain: { 
+        type: "boolean", 
+        describe: 'Flag to connect existing running holochain process'
+      },
+      relativePath: { 
```

Comment:
How about use `dataPath`, description to something like: `The relative path for storing ad4m data`?